flink/v1.18/flink/src/main/java/org/apache/iceberg/flink/source/IcebergTableSource.java

```diff
@@ -131,16 +131,21 @@ private DataStream<RowData> createDataStream(StreamExecutionEnvironment execEnv)
   private DataStreamSource<RowData> createFLIP27Stream(StreamExecutionEnvironment env) {
     SplitAssignerType assignerType =
         readableConfig.get(FlinkConfigOptions.TABLE_EXEC_SPLIT_ASSIGNER_TYPE);
-    IcebergSource<RowData> source =
+    // null check on assignerType
```

Comment:
I think this comment should be reworded or removed, but after a month nobody will understand what this supposed to mean 😄

Reply:
will remove it